packages/core/src/types/IVolumeInput.ts

```diff
@@ -10,6 +10,7 @@ type VolumeInputCallback = (params: {
   volumeActor: VolumeActor;
   /** unique volume Id in the cache */
   volumeId: string;
+  preset?;
```

Comment:
You don't need to put this here, the callback isn't getting a preset, it is using the `preset` from the surrounding